config/locales/fr.yml

```diff
@@ -36,15 +37,18 @@ fr:
     loading: Chargement en cours...
     location: Lieu
     log_in: Se connecter
+    min_read: min lues
     moderator_center: Centre de modération
     month: Mois
     more_from: Plus de la part de
     my_tags: Mes sujets
+    other: Autre
     podcasts: Podcasts
     pinned: Épinglé
     popular_tags: Sujets populaires
     post: Publication
     posts: Publications
+    posted_on: Posté sur
```

Comment:
This translation seems not reflected on:
https://github.com/forem/forem/blob/43adc9f1fbe5667340dca39cc7dcd50741d7e486/config/locales/views/articles/fr.yml#L84